### DISCUSSION TITLE
QMarkdown: UnhandledPromiseRejectionWarning in /app/lib/app-extension/Extension.js

### DISCUSSION THREAD

**Discussion by Dirk-** [quasarframework/quasar#12540] [Quasar App Extensions] [answered]
**EDIT: Sorry, please ignore this question!** I was away from Quasar development for quite a while and confused `quasar run` with `quasar dev`. Very stupid.

I ran `quasar upgrade -i` to upgrade:

```
 @quasar/extras: 1.12.2 → 1.12.5
 quasar: 2.3.4 → 2.5.5
 @quasar/app: 3.2.5 → 3.3.3
 @quasar/quasar-app-extension-qmarkdown: 2.0.0-alpha.9 → 2.0.0-beta.2
 eslint-plugin-quasar: 1.0.0 → 1.1.0

```
Now, `quasar run` gives me an error and does not start:

```
(node:15718) UnhandledPromiseRejectionWarning:   TypeError: Cannot read property 'charAt' of undefined
  
  - Extension.js:106 new Extension
    [whsm-frontend]/[@quasar]/app/lib/app-extension/Extension.js:106:14
  
  - quasar-run:51 run
    [whsm-frontend]/[@quasar]/app/bin/quasar-run:51:21

...
```  

I am not really sure this is caused by QMarkdown, but it is the only app extension I use. The project stems from Quasar v1, and I use QMarkdown since then, but it used to work in v2 before the recent upgrade.

Using Quasar CLI, I uninstalled QMarkdown and installed it again, but no change. Also, the `quasar.conf.js` file remains untouched and misses the parts which are described on the page [Installation Types](https://qmarkdown.netlify.app/all-about-qmarkdown/installation-types) of the QMarkdown documentation.

What can I do?

**Comment by MilosPaunovic** [comment, marked as answer]
You could always start by removing `package-lock.json` and `node_modules`, followed by a fresh dependency install with `npm i` and check if that helps.

**Reply by Dirk-** [reply]
Thanks, I did that before posting my question, but my error was much more stupid (see my edited question ;-)
